test/test_waveform.py

```diff
@@ -34,9 +34,8 @@
 
 _scheme, _context = parse_args_all_schemes("Waveform")
 
-failing_wfs = ['PhenSpinTaylor', 'PhenSpinTaylorRD', 'EccentricTD',
-              'SpinDominatedWf', 'EOBNRv2HM_ROM', 'EOBNRv2_ROM', 'EccentricFD',
-              'NR_hdf5', 'TEOBResum_ROM', 'Lackey_Tidal_2013_SEOBNRv2_ROM']
+# We only check a few as some require auxiliary files
+good_waveforms = ['IMRPhenomD', 'TaylorF2']
```

Comment:
Is it worth keeping some of the SpinTaylorT ones?

Reply:
I've added in SpinTaylorT5, and also an EOB one SEOBNRv4 (as that doesn't require a ROM file).